automation/deploy_igz/deploy_remote.py

```diff
@@ -0,0 +1,271 @@
+                live=True,
+            )
+
+        logger.info("Pushing mlrun-api docker image")
+
+        self._exec_local(
+            [
+                "docker",
+                "push",
+                built_image,
+            ],
+            live=True,
+        )
+
+        policy = """'{"spec":{"template":{"spec":{"containers":[{"name":"mlrun-api","imagePullPolicy":"Always"}]}}}}'"""
+        node = nodes[0]
+        self._connect_to_node(node)
+        try:
+            logger.info("Change mlrun-api-chief pull policy")
+            self._exec_remote(
+                [
+                    "kubectl",
+                    "-n",
+                    "default-tenant",
+                    "patch",
+                    "deployment",
+                    "mlrun-api-chief",
+                    "-p",
+                    policy,
+                ]
+            )
+            logger.info("Change mlrun-api-worker pull policy")
+            self._exec_remote(
+                [
+                    "kubectl",
+                    "-n",
+                    "default-tenant",
+                    "patch",
+                    "deployment",
+                    "mlrun-api-worker",
+                    "-p",
+                    policy,
+                ]
+            )
+
+            logger.info("Replace mlrun-api-chief")
+            self._exec_remote(
+                [
+                    "kubectl",
+                    "-n",
+                    "default-tenant",
+                    "set",
+                    "image",
+                    "deployment/mlrun-api-chief",
+                    f"mlrun-api={built_image}",
+                ]
+            )
+
+            logger.info("Replace mlrun-api-worker")
+            self._exec_remote(
+                [
+                    "kubectl",
+                    "-n",
+                    "default-tenant",
+                    "set",
+                    "image",
+                    "deployment/mlrun-api-worker",
+                    f"mlrun-api={built_image}",
+                ]
+            )
+
+            logger.info("Waiting for mlrun-api to become ready")
+            self._exec_remote(
+                [
+                    "kubectl",
+                    "-n",
+                    "default-tenant",
+                    "wait",
+                    "pods",
+                    "-l",
+                    "app.kubernetes.io/name=mlrun",
+                    "--for",
+                    "condition=Ready",
+                    "--timeout=300s",
+                ],
+                live=True,
+            )
+        finally:
+            self._disconnect_from_node()
+
+        logger.info("Deployed branch successfully! Yay!")
+
+
+@click.command(help="mlrun-api deployer to remote system")
+@click.option("--verbose", is_flag=True, help="Print what we are doing")
+@click.option(
+    "-c",
+    "--config",
+    help="Config file",
+    default="automation/deploy_igz//deploy_env.yml",
```

Comment:
```suggestion
    default="automation/deploy_igz/deploy_env.yml",
```
(although it's equivalent on POSIX)